public/app/plugins/panel/radialbar/suggestions.ts

```diff
@@ -1,101 +1,82 @@
-import { VisualizationSuggestionsBuilder } from '@grafana/data';
-import { FieldColorModeId } from '@grafana/schema';
+import { defaultsDeep } from 'lodash';
+
+import { FieldColorModeId, FieldType, VisualizationSuggestion, VisualizationSuggestionsHandler } from '@grafana/data';
+import { t } from '@grafana/i18n';
 import { GraphFieldConfig } from '@grafana/ui';
-import { SuggestionName } from 'app/types/suggestions';
 
 import { Options } from './panelcfg.gen';
 
-export class GaugeSuggestionsSupplier {
-  getSuggestionsForData(builder: VisualizationSuggestionsBuilder) {
-    const { dataSummary } = builder;
-
-    if (!dataSummary.hasData || !dataSummary.hasNumberField) {
-      return;
-    }
+export const radialBarSuggestionsHandler: VisualizationSuggestionsHandler<Options, GraphFieldConfig> = (
+  dataSummary
+) => {
+  if (!dataSummary.hasData || !dataSummary.hasFieldType(FieldType.number)) {
+    return;
```

Comment:
the radialbar needs to be able to set pluginId to 'gauge' for the feature toggle / suggestions to work